src/main/java/org/skriptlang/skript/bukkit/input/InputModule.java

```diff
@@ -0,0 +1,42 @@
+package org.skriptlang.skript.bukkit.input;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.EnumClassInfo;
+import ch.njol.skript.registrations.Classes;
+import ch.njol.skript.registrations.EventValues;
+import ch.njol.skript.util.Getter;
+import org.bukkit.event.player.PlayerInputEvent;
+
+import java.io.IOException;
+
+public class InputModule {
+
+	public static void load() throws IOException {
+		if (!Skript.classExists("org.bukkit.Input"))
+			return;
+
+		Skript.getAddonInstance().loadClasses("org.skriptlang.skript.bukkit.input", "elements");
```

Comment:
```suggestion
		Skript.getAddonInstance().loadClasses("org.skriptlang.skript.bukkit.input.elements");
```
You can actually just do this

Reply:
I just followed what was done in the other modules
[DisplayModule](https://github.com/SkriptLang/Skript/blob/dev/feature/src/main/java/org/skriptlang/skript/bukkit/displays/DisplayModule.java)
[BreedingModule](https://github.com/SkriptLang/Skript/blob/dev/feature/src/main/java/org/skriptlang/skript/bukkit/breeding/BreedingModule.java)

Reply:
Ah, those could be changed too I suppose. You can change it or keep it as-is. I doubt there is any sort of major performance difference.